_topic_maps/_topic_map.yml

```
@@ -25,6 +25,8 @@ Topics:
   File: ossm-running-v2-same-cluster-as-v3-assembly
 - Name: Red Hat OpenShift Service Mesh and cert-manager
   File: ossm-cert-manager-assembly
+- Name: Multi-Cluster Topologies
```

Capital "Cluster" or no? Seems only 3 hits for capitalized, many more for lowercase.

Also lowercase "topologies" for sentence-case headings.
```suggestion
- Name: Multi-cluster Topologies
```

---

I think it should be Multi-Cluster when it's first word in a sentence or first word in a title. I'll look at it and try to make them consistent. 

Changed to topologies for heading